### DISCUSSION TITLE
which wallet can support walletconnect v2  at Android ?

### DISCUSSION THREAD

**Discussion by KayCm** [WalletConnect/walletconnect-monorepo#2243] [Developer Support]
i had test so many wallets and none of them worked... and even the official demo wallet couldn't redirect back to my application

Does anyone know which wallet I cant connect with Walletconnect V2?

![Screenshot_20230420-172601_Android System](https://user-images.githubusercontent.com/7651323/233323312-6db8e274-91d2-48f4-967a-138a53dbd0a2.jpg)

![Screenshot_20230420-172622_One UI Home](https://user-images.githubusercontent.com/7651323/233323345-4bc6f08c-00cd-420d-8cdb-02f38cc25771.jpg)

**Comment by Talhaali00** [comment]
Hello @KayCm,
Can you explain how you are testing the redirect when using the demo app? 
Also, you can find a list of V2 compatible Wallets `[here](https://explorer.walletconnect.com/?type=wallet&version=2)`
